minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java

```diff
@@ -103,7 +88,7 @@ private MesosCluster(String clusterId) {
         this.clusterId = clusterId;
         this.clusterConfig = new ClusterConfig();
 
-        List<Container> dockerContainers = dockerClient.listContainersCmd().exec();
+        List<Container> dockerContainers = DockerClientFactory.get().listContainersCmd().exec();
```

Comment:
* should we have only one `DockerClient dockerClient = DockerClientFactory.get()` call per method and then re-use the local variable?
* what is the difference between `DockerClientFactory.get()` and the previous `DockerClientFactory.build()`?